src/components/Drawer.js

```diff
@@ -0,0 +1,62 @@
+const React = require('react');
+const Velocity = require('velocity-animate');
+
+const Button = require('../components/Button');
+
+const Drawer = React.createClass({
+  propTypes: {
+    onClose: React.PropTypes.func.isRequired
+  },
+
+  componentDidMount () {
+    const el = this._container;
+    const transition = { right: 0 };
+    const options = {
+      duration: 500
+    };
+
+    Velocity(el, transition, options);
```

Comment:
This looks very similar to how I did the PieChart component where the chart animates on load. React is not aware of this at all. There is no way to animate `onload` transitions after the component is mounted.